src/toil/batchSystems/options.py

```diff
@@ -106,10 +106,10 @@ def addOptions(addOptionFn, config):
     addOptionFn("--batchSystem", dest="batchSystem", default=defaultBatchSystem(),
                 help=("The type of batch system to run the job(s) with, currently can be one "
                       "of %s'. default=%s" % (', '.join(uniqueNames()), defaultBatchSystem())))
-    addOptionFn("--disableHotDeployment", dest="disableHotDeployment", action='store_true', default=None,
-                help=("Should hot-deployment of the user script be deactivated? If True, the user "
-                      "script/package should be present at the same location on all workers. "
-                      "default=false"))
+    addOptionFn("--disableRemoteDeployment", "--disableHotDeployment", dest="disableRemoteDeployment",
+                action='store_true', default=None,
+                help=("Remote-deployment=True copies the toil virtualenv and user script to the same location "
+                "on all workers automatically.  The default is False."))
```

Comment:
we should leave in --disableHotDeployment as a deprecated option (it can have dest=disableRemoteDeployment) and comment that it is deprecated. 

For API functions, we should put in deprecated ones that call the new ones (change the comments to fooHotDeploy is deprecated, use fooRemoveDeploy

Reply:
The old option should not be left in as-is.  Change the help to just say `Deprecated, use --disableRemoteDeployment instead' and have dest="disableRemoteDeployment"